xml/containers-get.xml

```diff
@@ -0,0 +1,586 @@
+     images</emphasis>. For example the label
+     <literal>com.suse.sle.base.title</literal> contains the image name "SUSE
+     Linux Enterprise 15 Base Container", which is information that cannot be
+     overwritten with any other layer of the image.
+    </para>
+    <para>
+     The opposite is true for {opensuse} base images, where labels do get
+     overwritten. For example <literal>org.opencontainers.image.title</literal>
+     can get overwritten and will always contain the title of the latest added
+     layer.
+    </para>
+   </important>
+   <para>
+    A list of all currently defined labels follows:
+   </para>
+   <table frame="all" rowsep="1" colsep="1">
+    <title>Currently Defined Labels</title>
+    <tgroup cols="5">
+     <colspec colname="col_1" colwidth="20*"/>
+     <colspec colname="col_2" colwidth="20*"/>
+     <colspec colname="col_3" colwidth="20*"/>
+     <colspec colname="col_4" colwidth="20*"/>
+     <colspec colname="col_5" colwidth="20*"/>
+     <thead>
+      <row>
+       <entry align="left" valign="top">Must be provided by derived images</entry>
+       <entry align="left" valign="top">Labels</entry>
+       <entry align="left" valign="top">OCI Annotation</entry>
+       <entry align="left" valign="top">Description</entry>
+       <entry align="left" valign="top">Example</entry>
+      </row>
+     </thead>
+     <tbody>
+      <row>
+       <entry align="left" valign="top">
+        <para>
+         ✓
+        </para>
+       </entry>
+       <entry align="left" valign="top">
+        <para>
+         org.opencontainers.image.<emphasis role="strong">title</emphasis>
+         com.suse.sle.base.<emphasis role="strong">title</emphasis>
+        </para>
+       </entry>
+       <entry align="left" valign="top">
+        <para>
+         org.opencontainers.image.title
+        </para>
+       </entry>
+       <entry align="left" valign="top">
+        <para>
+         Title of the image
+        </para>
+       </entry>
+       <entry align="left" valign="top">
+        <para>
+         SUSE Linux Enterprise 15 Base Container
+        </para>
+       </entry>
+      </row>
+      <row>
+       <entry align="left" valign="top">
+        <para>
+         ✓
+        </para>
+       </entry>
+       <entry align="left" valign="top">
+        <para>
+         org.opencontainers.image.<emphasis role="strong">description</emphasis>
+         com.suse.sle.base.<emphasis role="strong">description</emphasis>
+        </para>
+       </entry>
+       <entry align="left" valign="top">
+        <para>
+         org.opencontainers.image.description
+        </para>
+       </entry>
+       <entry align="left" valign="top">
+        <para>
+         Short description of the image
+        </para>
+       </entry>
+       <entry align="left" valign="top">
+        <para>
+         Image containing a minimal environment for containers based on SUSE
+         Linux Enterprise 15.
```

Comment:
Use an entity instead of bare SUSE Linux Enterprise? (Not completely sure because of the context.)